ragtime.core/src/ragtime/main.clj

```diff
@@ -27,16 +27,34 @@
   (map verbose-migration ((load-var migration-fn))))
 
 (defn migrate [{:keys [database migrations]}]
-  (core/migrate-all
-   (core/connection database)
-   (resolve-migrations migrations)))
+  (letfn [(migrate-db [the-db]
+                      (core/migrate-all
+                        (core/connection the-db)
+                        (resolve-migrations migrations))
+                      )
+          ]
+    (if (coll? database)
+      (doseq [db database]
+        (migrate-db db)
+        )
+      (migrate-db database)
+      )))
```

Comment:
I don't think the rest of the codebase leaves trailing parentheses hanging on a separate line.